yamcs-core/src/main/java/org/yamcs/tctm/ccsds/AbstractTmFrameLink.java

```diff
@@ -20,6 +20,7 @@ public abstract class AbstractTmFrameLink extends AbstractLink implements Aggreg
     protected List<Link> subLinks;
     protected MasterChannelFrameHandler frameHandler;
     protected AtomicLong frameCount = new AtomicLong(0);
```

Comment:
one small change if you don't mind, lets call this validFrameCount so then it's 100% clear.

Reply:
Done (you may have seen that I originally misinterpreted your original comment to mean that :P)

Reply:
ah, ok, then why not squash the three commits into one and we are done with it.

thank you very much for your work :)